docs/notebooks/extended_demo.py

```diff
@@ -1 +1,188 @@
+        "allowable_networks": ["drive"],
+        "omit_edges": ["bridge"],
+    },
+    "outfall_derivation": {
+        "outfall_length": 5,
+        "river_buffer_distance": 30,
+    },
+}
+outputs = swmmanywhere(config)
+plot_map(outputs[0].parent)
+
+# %% [markdown]
+# OK that clearly helped, although we have appear to have stranded pipes (e.g., along
+# *Carrer dels Canals* in North West), presumably due to some mistake in the
+# OSM specifying that it
+# is connected via a pedestrian route. We won't remedy this in the tutorial, but you can
+# manually provide your
+# [`starting_graph`](https://imperialcollegelondon.github.io/SWMManywhere/config_guide/#change-starting_graph)
+# via the configuration file to address such mistakes.
+#
+# More importantly we can see some distinctive unconnected network in the South West.
+# What is going on there? To explain this we will have to turn on verbosity to print the
+# intermediate files used in model derivation.
+#
+# To do this with a command line call we simply add the flag `--verbose=True`.
+# Though in code we will have to use `set_verbose` from the `logging` module.
+
+# %%
+# Make verbose
+set_verbose(True)  # Set verbosity
+
+# Run again
+outputs = swmmanywhere(config)
+model_dir = outputs[0].parent
+m = plot_map(model_dir)
+
+# %% [markdown]
+# That's a lot of information! However, the reason we are currently interested
+# in this is because the files associated with
+# each workflow step are saved when `verbose=True`.
+#
+# We will load a file called `subbasins` and add it to the map.
+
+# %%
+subbasins = gpd.read_file(model_dir / "subbasins.geoparquet")
+folium.GeoJson(subbasins, fill_opacity=0, color="blue", weight=2).add_to(m)
+m
+
+# %% [markdown]
+# Although this can be customised, the default behaviour of `swmmanywhere` is to not
+# allow edges to cross hydrological subbasins. It is now super clear why these
+# unconnected  networks have appeared, and are ultimately due to the underlying DEM.
+# If you did desperately care about these streets, then you should probably
+# widen your bounding box.
+#
+# ## Plotting results
+#
+# Because we have run the model with `verbose=True` we will also see that a new
+# `results` file has appeared, which contains all of the simulation results from SWMM.
+
+# %%
+df = pd.read_parquet(model_dir / "results.parquet")
+df.head()
+
+# %% [markdown]
+# `results` contains all simulation results in long format, with `flooding` at
+# nodes and `flow` at edges. We will plot a random `flow`.
+
+# %%
+flows = df.loc[df.variable == "flow"]
+flows.loc[flows.id == flows.iloc[0].id].set_index("date").value.plot(
+    ylabel="flow (l/s)"
+)
+
+
+# %% [markdown]
+# If `results` are present in the `model_dir`, `plot_map` will make clickable
+# elements to view plots,
+# now you can inspect your results in a much more elegant way than the SWMM GUI.
+# Just click a node or link to view the flooding or flow timeseries!
+
+
+# %%
+plot_map(model_dir)
```

Comment:
I'm getting negative flows here. Does it make sense?

<img width="402" alt="image" src="https://github.com/user-attachments/assets/a021ab2a-83c2-41bb-b867-609190ea8685">

Reply:
It can happen - flows are 'often' (i.e., but not always) sensible ;) mainly due to the starting graph as the text says

Reply:
I will update the text to mention it